.gitignore

```diff
@@ -130,3 +130,5 @@ vendor/bundle/*
 
 # ignore auto-generated plugin js assets
 /app/assets/javascripts/plugins/*
+yarn-error.log
+.rubocop-https---raw-githubusercontent-com-discourse-*
```

Comment:
This shouldn't be needed anymore. That file was a byproduct of the old method of including the rubocop config. (via the URL)

Reply:
yeh right, I should remove it from my generator